src/main/java/training/subsystems/Wrist/WristConstants.java

```diff
@@ -0,0 +1,12 @@
+package training.subsystems.Wrist;
+
+public class WristConstants {
+    //* These constants are temp and shall be replaced with calibrated constants that aren't 0
+    static final String WRIST_LOG_PATH = "";
+    static final int WRIST_ID = 0;
+
+    static final double POWER_LIMIT_WRIST = 0.9; // limit both min and max
+    static final int PEAK_DURATION_WRIST_MS = 10; // time for PEAK_MAX_CURRENT_AMP to take effect before using the continues val
```

Comment:
no comments, find better name and dont use comments

Reply:
it means Mi€®Ø$0₣ŧ™